example/model-parallel/matrix_factorization/README.md

```diff
@@ -0,0 +1,21 @@
+Model Parallel Matrix Factorization
+===================================
+
+The example walks you through a matrix factorization algorithm for recommendations and also
```

Comment:
The example => This example.
Will be helpful to link the paper for matrix factorization algorithm